index.css

```diff
@@ -78,3 +105,78 @@ label {
     color: rgb(60, 162, 178);
 }
 
+h1 {
+        height: 50px;
+        text-align: center;
+    }
+  
+    h1 span {
+        position: relative;
+        top: 20px;
+        display: inline-block;
+        animation: bounce .7s ease infinite alternate;
+        font-family: 'Titan One'cursive;
+        font-size: 50px;
+        color: rgb(153, 216, 53);
+        text-shadow: 0 1px 0 green,
+            0 2px 0 green,
+            0 3px 0 green,
+            0 4px 0 green,
+            0 5px 0 green,
+            0 6px 0 transparent,
+            0 7px 0 transparent,
+            0 8px 0 transparent,
+            0 9px 0 transparent,
+            0 10px 10px rgba(0, 0, 0, 0.4);
+    }
+  
+    h1 span:nth-child(2) {
+        animation-delay: .1s;
+    }
+  
+    h1 span:nth-child(3) {
+        animation-delay: .2s;
+    }
+  
+    h1 span:nth-child(4) {
+        animation-delay: .3s;
+    }
+  
+    h1 span:nth-child(5) {
+        animation-delay: .4s;
+    }
+  
+    h1 span:nth-child(6) {
+        animation-delay: .5s;
+    }
+  
+    h1 span:nth-child(7) {
+        animation-delay: .6s;
+    }
+  
+    h1 span:nth-child(8) {
+        animation-delay: .7s;
+    }
+    h1 span:nth-child(9) {
+        animation-delay: .8s;
+    }
+    h1 span:nth-child(10) {
+        animation-delay: .9s;
+    }
+
+  
+    @keyframes bounce {
+        100% {
+            top: -20px;
+            text-shadow: 0 1px 0 #CCC,
+                0 2px 0 #CCC,
+                0 3px 0 #CCC,
+                0 4px 0 #CCC,
+                0 5px 0 #CCC,
+                0 6px 0 #CCC,
+                0 7px 0 #CCC,
+                0 8px 0 #CCC,
+                0 9px 0 #CCC,
+                0 50px 25px rgba(0, 0, 0, 0.2);
+        }
+    }
```

Comment:
Nice to see your work in css! Just out of curiosity, where did you learn how to do this?

Reply:
Thanks, Leon. I learn it from our good friend 'Google'  :D 
I just searched and found these,
https://www.geeksforgeeks.org/how-to-create-jumping-text-3d-animation-effect-using-css/
https://www.geeksforgeeks.org/how-to-create-fade-in-effect-on-page-load-using-css/